### PR TITLE
Create categories page

### PR DESCRIPTION
**Pull request by simonychuang**
Adds first iteration of page in core to manage content categories. Allows basic CRUD operation with only the `name` field for now. Similar to Assets page categories management. UI will be improved in the next cycle, right now it's a very barebones implementation just to get the functionality in.

/nocl

## How Has This Been Tested?

Tested manually on local env

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: graylog2-web-interface/src/components/navigation/bindings.ts [hunk 1]
```diff
@@ -63,6 +63,7 @@ const navigationBindings: PluginExports = {
             { path: Routes.SYSTEM.LOOKUPTABLES.OVERVIEW, description: 'Lookup Tables', permissions: ['lookuptables:read'] },
             { path: Routes.SYSTEM.PIPELINES.OVERVIEW, description: 'Pipelines', permissions: ['pipeline:read', 'pipeline_connection:read'] },
             { path: Routes.SYSTEM.SIDECARS.OVERVIEW, description: 'Sidecars', permissions: ['sidecars:read'] },
+            { path: Routes.SYSTEM.ENTITYGROUPS.OVERVIEW, description: 'Manage Entity Groups' },
```
Comment (ryan-carroll-graylog) [inline graylog2-web-interface/src/components/navigation/bindings.ts:66]: Seems to fit the pattern of the other items here to just have `Entity Groups`, with `Manage` being implied?
```suggestion
            { path: Routes.SYSTEM.ENTITYGROUPS.OVERVIEW, description: 'Entity Groups' },
```

**Review by ryan-carroll-graylog — COMMENTED** [review]
Basic functionality looks solid! I know there was talk about having the basic ability to add an entity id/type to a group, i.e. 2 more fields that take a type string/id combo and add to add to a group. But I don't think it's super important at this stage if there's other cycle work.

Just left one other comment about possibly tweaking the menu item name.

File: graylog2-web-interface/src/components/navigation/bindings.ts (hunk 1 above)
Comment (simonychuang) [reply graylog2-web-interface/src/components/navigation/bindings.ts:66]: @ryan-carroll-graylog Yup, that thought definitely crossed my mind but without any mockups at the moment and with the plan to continue working on this next cycle I decided to leave out the fields to add entities to a group for now.

Comment (ryan-carroll-graylog) [reply graylog2-web-interface/src/components/navigation/bindings.ts]: Ah ok definitely makes sense to me to wait for mockups. The API is pretty simple for anyone wanting to use this as is anyway.

**Review by ryan-carroll-graylog — APPROVED** [review]

**Pull request merged**